.travis.yml

```diff
@@ -23,7 +23,23 @@ matrix:
           packages: ['g++-6']
       env: COMPILER='g++-6'
 
-#    # 3/ OSX Clang Builds - disabled for now
+    # 3/ Linux GCC 4.9 Builds
```

Comment:
version 4.9 is yet a Linux GCC Build, isn't it?

Reply:
I'm not sure I understand.